dexs/syncswap/index.ts

```diff
@@ -0,0 +1,18 @@
+import { univ2Adapter } from "../../helpers/getUniSubgraphVolume";
+import { CHAIN } from "../../helpers/chains";
+
+const endpoints = {
+  [CHAIN.ERA]: "https://api.studio.thegraph.com/query/30365/zksync-blocks/v0.0.9",
+};
+
+const adapter = univ2Adapter(endpoints, {
+  factoriesName: "syncSwapFactories",
+  dayData: "syncSwapSwitchDayData",
```

Comment:
change "syncSwapSwitchDayData" to "dayData"

Reply:
Thanks, good catch